build-farm/platform-specific-configurations/windows.sh

```diff
@@ -33,16 +33,16 @@ if [ "$ARCHITECTURE" == "aarch64" ]; then
 fi
 BOOT_JDK_VARIABLE="JDK${JDK_BOOT_VERSION}_BOOT_DIR"
 if [ ! -d "$(eval echo "\$$BOOT_JDK_VARIABLE")" ]; then
-  bootDir="$PWD/jdk$JDK_BOOT_VERSION"
+  bootDir="$PWD/jdk-$JDK_BOOT_VERSION"
```

Comment:
PR says to remove hyphen but this change is adding?

Reply:
Ive reversed the windows changes, see my comment here https://github.com/adoptium/temurin-build/pull/3992#issuecomment-2457395642